### DISCUSSION TITLE
How can I recolour only a single square or a squareset?

### DISCUSSION THREAD

**Discussion by FowadIjaz** [niklasf/python-chess#820] [Q&A]
Hi, just wondering how it's possible to recolour only a single square to a different colour. In the docs, I can see that chess.svg.board() has an argument 'colors' which can take a dictionary and recolour for instance all light squares or all dark squares, but if I want to select a single square or squareset and recolour that, how would I go about it? Thanks in advance.

**Comment by niklasf** [comment]
Hi. See https://github.com/niklasf/python-chess/issues/774#issuecomment-826378529.
